### ISSUE TITLE
Starting up minecraft issue

### ISSUE THREAD

**Issue by battery3996** [iChun/ContentCreatorIntegration-IssuesAndDocumentation#46]
MC: 1.15.2
Forge: recommended 31.1.0 and latest 31.1.48
CCI: 1.5.0
Testing the new CCI, i have been unable to get to the main menu. Starting it up with the console open i noticed that it is giving errors. 
```java
Failed to create mod instance. ModID: contentcreatorintegration, class me.ichun.mods.cci.common.ContentCreatorIntegration
java.lang.NullPointerException
	at me.ichun.mods.cci.common.event.EventHandler.readConstants(EventHandler.java:101)
	at me.ichun.mods.cci.common.event.EventHandler.init(EventHandler.java:38)
	at me.ichun.mods.cci.common.ContentCreatorIntegration.lambda$null$2(ContentCreatorIntegration.java:111)
	at me.ichun.mods.cci.common.ContentCreatorIntegration$$Lambda$2235/1577337346.run(Unknown Source)
	at net.minecraftforge.fml.DistExecutor.runWhenOn(DistExecutor.java:56)
	at me.ichun.mods.cci.common.ContentCreatorIntegration.<init>(ContentCreatorIntegration.java:105)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at java.lang.Class.newInstance(Class.java:442)
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:131)
	at net.minecraftforge.fml.javafmlmod.FMLModContainer$$Lambda$2155/1863410679.accept(Unknown Source)
	at java.util.function.Consumer.lambda$andThen$14(Consumer.java:65)
	at java.util.function.Consumer$$Lambda$2154/602552403.accept(Unknown Source)
	at java.util.function.Consumer.lambda$andThen$14(Consumer.java:65)
	at java.util.function.Consumer$$Lambda$2154/602552403.accept(Unknown Source)
	at net.minecraftforge.fml.ModContainer.transitionState(ModContainer.java:112)
	at net.minecraftforge.fml.ModList.lambda$null$10(ModList.java:134)
	at net.minecraftforge.fml.ModList$$Lambda$2216/289080492.accept(Unknown Source)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1374)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:512)
	at java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:291)
	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinTask.doInvoke(ForkJoinTask.java:401)
	at java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:734)
	at java.util.stream.ForEachOps$ForEachOp.evaluateParallel(ForEachOps.java:160)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateParallel(ForEachOps.java:174)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233)
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:583)
	at net.minecraftforge.fml.ModList.lambda$dispatchParallelEvent$11(ModList.java:134)
	at net.minecraftforge.fml.ModList$$Lambda$2204/1911860918.run(Unknown Source)
	at java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1386)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1689)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
18:30:03.508
Failed to complete lifecycle event CONSTRUCT, 1 errors found
18:30:03.510
EventBus 0 shutting down - future events will not be posted.
java.lang.Exception: stacktrace
	at net.minecraftforge.eventbus.EventBus.shutdown(EventBus.java:278)
	at net.minecraftforge.fml.client.ClientModLoader.lambda$createRunnableWithCatch$5(ClientModLoader.java:115)
	at net.minecraftforge.fml.client.ClientModLoader$$Lambda$2039/808663361.run(Unknown Source)
	at net.minecraftforge.fml.client.ClientModLoader.begin(ClientModLoader.java:97)
	at net.minecraft.client.Minecraft.<init>(Minecraft.java:393)
	at net.minecraft.client.main.Main.main(SourceFile:166)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:56)
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider$$Lambda$385/1967617997.call(Unknown Source)
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37)
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54)
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72)
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:81)
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:65)
```

I also noticed that a similar issue was open up. In the comments They posted to make sure that the configs has "" for the streamlabs key and twitch username. I have that in the config yet it still has this issue.
https://i.imgur.com/HSI02VE.png

**Comment by battery3996** [comment]
Ok i found a temporary fix for this.
I launched mc with an olderversion of CCI and it booted up. Updated to the latest cci and now it boots.

**Comment by iChun** [comment]
Fixed in 1.5.1
